InitialCommit

```diff
@@ -0,0 +1,112 @@
+var sport = prompt("Time to get coaching advice on a sport!");
```

Comment:
Don't think this variable is being used anywhere, might be a better use case for `alert`